src/mixins/repetition.js

```diff
@@ -24,44 +28,79 @@ export const RepetitionMixin = (superclass) =>
             if (repetitionsAlongLatticeVectorB < maxNumberOfRepetitions)
-                columns = columns.filter((item, index) => index%maxNumberOfRepetitions < repetitionsAlongLatticeVectorB)
+                columns = columns.filter(
+                    (item, index) =>
+                        index % maxNumberOfRepetitions < repetitionsAlongLatticeVectorB,
+                );
             if (repetitionsAlongLatticeVectorC < maxNumberOfRepetitions)
-                columns = columns.map(arr => arr.filter((item, index) => index < repetitionsAlongLatticeVectorC))
+                columns = columns.map((arr) =>
+                    arr.filter((item, index) => index < repetitionsAlongLatticeVectorC),
+                );
             return columns.reduce((res, item) => {
-                res.push(...item)
-                return res
-            }, [])
+                res.push(...item);
+                return res;
+            }, []);
         }
 
         /**
          * Repeats a given 3D object at the lattice points given by repetitionCoordinates function.
          */
         repeatObject3DAtRepetitionCoordinates(object3D) {
-            const { settings: {repetitionsAlongLatticeVectorA, repetitionsAlongLatticeVectorB, repetitionsAlongLatticeVectorC} } = this
-            const coordinates = this.repetitionCoordinates(Math.max(repetitionsAlongLatticeVectorA, repetitionsAlongLatticeVectorB, repetitionsAlongLatticeVectorC));
+            const {
+                settings: {
+                    repetitionsAlongLatticeVectorA,
+                    repetitionsAlongLatticeVectorB,
+                    repetitionsAlongLatticeVectorC,
+                },
+            } = this;
+            const coordinates = this.repetitionCoordinates(
+                Math.max(
+                    repetitionsAlongLatticeVectorA,
+                    repetitionsAlongLatticeVectorB,
+                    repetitionsAlongLatticeVectorC,
+                ),
+            );
             this.structureGroup.add(object3D);
-            this.coordinatesByAxes(coordinates, {repetitionsAlongLatticeVectorA, repetitionsAlongLatticeVectorB, repetitionsAlongLatticeVectorC}).slice(1).forEach((point) => {
-                const object3DClone = object3D.clone();
-                object3DClone.position.add(new THREE.Vector3(...point));
-                this.structureGroup.add(object3DClone);
-            });
+            this.coordinatesByAxes(coordinates, {
+                repetitionsAlongLatticeVectorA,
+                repetitionsAlongLatticeVectorB,
+                repetitionsAlongLatticeVectorC,
+            })
+                .slice(1)
+                .forEach((point) => {
+                    const object3DClone = object3D.clone();
+                    object3DClone.position.add(new THREE.Vector3(...point));
+                    this.structureGroup.add(object3DClone);
+                });
```

Comment:
The image file below should have Silicon atom at [0, 0, 0] (in A, B, C crystal coordinates) and not at [1, 1, 1] as it has now

Reply:
Here we can check if our move functionality works well. If I make a basic model - we will not see the result